test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -769,6 +773,19 @@ def setUp(self):
     e.reject("Cannot resize the job")
 """
 
+    def is_cgroup(self, host):
+        """
+        Returns true if cgroup is mounted on the mom host, false otherwise.
+        """
+        ret = self.du.cat(host, '/proc/mounts')
+        val = False
+        for mnt in ret['out']:
+            if 'cgroup' not in mnt:
+                continue
+            else:
+                val = True
+        return val
+
```

Comment:
This function tells us if one or more cgroups are mounted, but doesn't tell us which ones. If blockio were the only cgroup mounted, the check would pass but the tests would fail because we don't use blockio. I think it should return a list of mounted cgroups so that a test can verify the ones it needs are mounted.

Reply:
I see what you mean.  get_paths() is useful for what you are saying but it seems to check locally only, not on each mom.  But it can be modified to check on each mom.